app/src/main/java/org/greenstand/android/TreeTracker/di/NetworkModule.kt

```diff
@@ -41,4 +42,17 @@ val networkModule = module {
             .create(ApiService::class.java)
     }
 
+    single(named("paymentApi")) {
```

Comment:
we probably may not need the qualifier when we have the final unified api that includes the payment data. For now, my assumption requires one existing endpoint to support existing integrations and the new planter account api for the payment and tree data.